stable/kube-state-metrics/templates/role.yaml

```diff
@@ -0,0 +1,184 @@
+{{- if and (eq $.Values.rbac.create true) (.Values.namespace) (eq .Values.rbac.useClusterRole false) (not .Values.rbac.useExistingRole)  -}}
```

Comment:
Could the clusterrole and this new role be merged, so we can apply DRY to the rbac permissions?

Reply:
I've made an attempt at that.. it generates the same yaml for my testcase (multiple namespaces and useClusterrole false)..